kubemarine/templates/plugins/calico-v3.21.yaml.j2

```diff
@@ -4103,7 +4103,7 @@ spec:
         cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
     spec:
       nodeSelector:
-        {{ plugins['calico']['typha']['nodeSelector'] | toyaml | indent(width=8, indentfirst=False) }}
+        {{ plugins['calico']['typha']['nodeSelector'] | toyaml | indent(width=8, first=False) }}
```

Comment:
Looks like we no longer use templates for calico at all.

Reply:
delete calico 3.21